18 key bluetooth NUMBERPAD (#9167)
qmk/qmk_firmware: keyboards/18bluetoothpad/readme.md

> 18bluetoothpad

Reviewer: ```suggestion
# 18bluetoothpad
```

Author: Approved

![18bluetoothpad](https://github.com/latincompass/18bluetoothpad/blob/master/ZZ20200421113009.png)

> ![18bluetoothpad](https://github.com/latincompass/18bluetoothpad/blob/master/IMG_0116.JPG)
> ![18bluetoothpad](https://github.com/latincompass/18bluetoothpad/blob/master/IMG_0117.JPG)

Reviewer: ```suggestion
```

Author: Approved

* Keyboard Maintainer: latincompass
* Hardware Availability: https://github.com/latincompass/18bluetoothpad

Make example for this keyboard (after setting up your build environment):

>     make 18bluetoothpad:

Reviewer: ```suggestion
    make 18bluetoothpad:default
```
oops

> Keyboard Maintainer: latincompass
> Hardware Availability: https://github.com/latincompass/18bluetoothpad
> Make example for this keyboard (after setting up your build environment):
> 
> make 18bluetoothpad:default
> See the build environment setup and the make instructions for more information. Brand new to QMK? Start with our Complete Newbs Guide.

Reviewer: ```suggestion
* Keyboard Maintainer: latincompass
* Hardware Availability: https://github.com/latincompass/18bluetoothpad

Make example for this keyboard (after setting up your build environment):

    make 18bluetoothpad:

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

Author: Approved
